task-list/app/assets/javascripts/home.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
When I got code reviews from C3 folks, they told me to delete files I don't use, like these coffescript one that are automatically generated by Rails. I never actually followed their advice but now I'm giving the same advice for you. I guess it doesn't matter for now, but is a good thing to keep in mind for later.